docs/decisions/uswds.md

```diff
@@ -0,0 +1,40 @@
+
+### Continuity
+
+A reasonable analogy to make here is [Bootstrap](https://getbootstrap.com/). Much as Bootstrap was once the de facto CSS framework used by countless websites, USWDS has positioned itself as the go-to styling and design solution for federal government websites. Whereas Bootstrap was maligned over time for its relatively rigid appearance, resulting in a profliferation of visually homogenous sites, we consider that same characteristic less of a negative for USWDS. One USWDS design principle is to [promote continuity](https://designsystem.digital.gov/design-principles/#promote-continuity-2), detailed with the broader goal to "minimize disruption, and provide a consistent experience throughout services; over time; and across agencies, platforms, and devices." Using USWDS places LDAF's site within a visual identity many will already be implicitly familiar with via interactions on other modern US government websites.
+
+Another angle to view continuity is internally, as many of our engineers and designers have experience working with USWDS. Beginning to build with familiar tooling has a few notable advantages:
+
+1. **We're not starting from scratch.** Designers are provided a [full set of design assets](https://designsystem.digital.gov/documentation/getting-started-for-designers/) available across a variety of file formats and platforms, and engineers have a large set of [common components](https://designsystem.digital.gov/components/overview/) as a starting point. While there's plenty of additional customization and tooling to add on top of the base components, its a strong pre-established foundation from which to begin.
+2. **Continuity can be a part of hand-offs and personnel changes, too.** There are plenty of people not on the LDAF team that are familiar with USWDS and have worked with it in varying capacities. This could mean one less area of context to build up when someone new is orienting themselves within the LDAF project architecture.
+
+### Accessibility
+
+Per USWDS' principle to **[Embrace Accessibility](https://designsystem.digital.gov/design-principles/#embrace-accessibility-2)**: "Accessibility affects everybody; build it into every decision." This goes beyond evalution criteria and is [baked into our wider ethos](https://adhoc.team/playbook-accessibility/). Given USWDS' excellent accessibility baked into it from the start, it makes our duty as designers and engineers to build everything with accessibility in the forefront that much easier.
+
+### Battle-Tested
+
+Lastly, while this isn't a direct design principle of USWDS, it's reached a point of maturity where it's relatively proven as a reliable and fully-featured solution. The ecosystem is such that we're confident it will be maintained and improved up in the future
```

Comment:
```suggestion
Lastly, while this isn't a direct design principle of USWDS, it's reached a point of maturity where it's relatively proven as a reliable and fully-featured solution. The ecosystem is such that we're confident it will be maintained and improved up in the future.
```